Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1108 @@
+display: Thycotic
+name: Thycotic
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved password from secret
+    execution: false
+    name: thycotic-secret-password-get
+    outputs:
+    - contextPath: Thycotic.Secret.Password
+      description: Current password from secret
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved username from secret
+    execution: false
+    name: thycotic-secret-username-get
+    outputs:
+    - contextPath: Thycotic.Secret.Username
+      description: Retrieves username from secret.
+      type: String
+  - arguments:
+    - default: false
+      description: Search for secret using secret name.
+      isArray: false
+      name: search_name
+      required: true
+      secret: false
+    deprecated: false
+    description: Search ID secret by Name
+    execution: false
+    name: thycotic-secret-search-name
+    outputs:
+    - contextPath: Thycotic.Secret.Id
+      description: Retrieves list of Secret IDs.
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    - default: false
+      description: New password value
+      isArray: false
+      name: newpassword
+      required: true
+      secret: true
+    deprecated: false
+    description: Update password for a secret by ID
+    execution: false
+    name: thycotic-secret-password-update
+    outputs:
+    - contextPath: Thycotic.Secret.Newpassword
+      description: New password changed for secret
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Check Out a Secret
+    execution: false
+    name: thycotic-secret-checkout
+    outputs:
+    - contextPath: Thycotic.Secret.Checkout
+      description: Returns response (None = Successful)
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Check In a secret
+    execution: false
+    name: thycotic-secret-checkin
+    outputs:
+    - contextPath: Thycotic.Secret.Checkin
+      description: Object Secret summary
+      type: String
+  - arguments:
+    - default: false
+      description: Folder name
+      isArray: false
+      name: foldername
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Folder type ID(1=< ID =< 3
```

Comment:
Could you please explain this description?

Reply:
@guykeller 
The description of this parameter is taken from the REST API documentation for Thycotic Secret Server.

`Available options for creating a secret folder

folderName: string 
Folder name

folderTypeId: integer (int32) , { x ∈ ℤ | 1 ≤ x ≤ 3 } 
Folder type ID

inheritPermissions: boolean
Whether the folder should inherit permissions from its parent (default: true)

inheritSecretPolicy: boolean
Whether the folder should inherit the secret policy. Defaults to true unless creating a root folder.

parentFolderId: integer (int32) , { x ∈ ℤ | x ≥ -1 } 
Parent folder ID

secretPolicyId: integer (int32)
Secret policy ID`

Reply:
In which case I would make this a predefined value list - 1, 2 or 3

Reply:
So it was done
![Screenshot from 2021-02-08 14-11-12](https://user-images.githubusercontent.com/69254946/107218001-afbb1e80-6a17-11eb-826f-68b23a1bfb07.png)